### DISCUSSION TITLE
1.20.x versions require Sodium 0.5.3 but latest version is 0.5.11

### DISCUSSION THREAD

**Discussion by Hexicube** [iPortalTeam/ImmersivePortalsMod#1656] [Compatibility Issue Report]
### If it's a mod compatibility issue, please provide the link(s) to the incompatible mod(s)

https://modrinth.com/mod/sodium
https://modrinth.com/mod/zenith

### Minecraft version

1.20.1

### Describe the issue

Because an older version of Sodium is required, I'm getting technically unrelated crashes in Zenith when reforging enchanted items due to a bug in Sodium that is fixed in the latest version.

It's technically possible to work around this crash by not reforging certain items whilst Sodium is active, however this is highly inconvenient.

**Comment by BlasterM5** [comment]
My Q too, except Sodium updated to 0.5.13 in 1.20.1. Someone made a compat called Immersive Portals Continued, but it hasn't been updated in a year (designed for 0.5.11), and there are ... issues. Reason to stay 1.20.1 is because Origins server, which is still 1.20.1 for stable (No, 1.20.2 is not viable due to Mojang temporarily removing some mcfunction capabilities), and reason to keep Sodium is because if you don't have an overpowered system, the framerates are awful without Sodium.
